server/src/db/schemas/compliance.js

```diff
@@ -0,0 +1,88 @@
+const mongoose = require('mongoose');
+
+// SCHEMA FORM CONTENT
+const ComplianceDetailSchema = new mongoose.Schema({
+    bbSpecification: {
+        type: String,
+        required: true
+    },
+    bbVersion: {
+        type: String,
+        required: true
+    },
+    status: {
+        type: String,
+        enum: ['in review', 'approved', 'rejected'],
```

Comment:
please change those enums to numeric, you can add enum class like 
```js
const StatusEnum = {
  IN_REVIEW: 1,
  APPROVED 2,
  REJECTED: 3
};
```
I also believe that we had also 0 for DRAFT status.

Reply:
Good point, I will double check all of the values if I have miss something. Draft should be present in this enum so I will add it